c/zss.c

```diff
@@ -1041,6 +1041,21 @@ static void readAgentAddressAndPort(JsonObject *serverConfig, JsonObject *envCon
   }
 }
 
+static char* generateCookieName(JsonObject *envConfig, int port) {
+  int cookieLength=256;
+  char *cookieName = safeMalloc(cookieLength+1, "CookieName");
+  char *zoweInstanceId = jsonObjectGetString(envConfig, "ZOWE_INSTANCE");
```

Comment:
`envConfig` contains only `ZWED`-prefixed environment variables. Shouldn't you get other env vars with `getenv()`?